deploy-manage/security/fips-ingest.md

```diff
@@ -0,0 +1,95 @@
+Be sure to enforce security in your FIPS environments through other means, such as strict file permissions and access controls on the key file itself, for example. 
+
+These TLS related restrictions apply to all components listed--{{agent}}, {{fleet}}, {{filebeat}}, {{metricbeat}}, and {{apm-server}}. 
+
+### General output and input limitations (Kerberos protocol) [ingest-inputoutput-limitations]
+
+The Kerberos protocol is not supported for any output or input, which also impacts the available `sasl.mechanism` for the Kafka output where only `PLAIN` is supported. 
+
+This impacts [Filebeat](beats://reference/filebeat/configuration-kerberos.md), [Metricbeat](beats://reference/metricbeat/configuration-kerberos.md) and {{apm-server}}, as well as output configurations for {{agent}} with {{fleet-server}}. 
+
+
+### APM Server [ingest-apm-limitations]
+
+* The [Secrets Keystore](/solutions/observability/apm/secrets-keystore-for-secure-settings.md) is not supported. 
+
+### Filebeat [ingest-filebeat-limitations]
+
+* The [Secrets Keystore](beats://reference/filebeat/keystore.md) is not supported. 
+* The [Translate GUID processor](beats://reference/filebeat/processor-translate-guid.md) is not supported.
+* The [Fingerprint processor](beats://reference/filebeat/fingerprint.md) does not support the md5 and sha1 method. 
+* The [Community ID Network Flowhash processor](beats://reference/filebeat/community-id.md) is not supported. 
+* The [Azure module](beats://reference/filebeat/filebeat-module-azure.md) including the [Azure eventhub input](beats://reference/filebeat/filebeat-input-azure-eventhub.md) and the [Office 365 module (Beta)](beats://reference/filebeat/filebeat-module-o365.md) is currently not supported. The [Add Cloud Metadata processor](beats://reference/filebeat/add-cloud-metadata.md) does not support the Azure Virtual Machine provider currently. 
+* The [GCP Pub/Sub input](beats://reference/filebeat/filebeat-input-gcp-pubsub.md) is not supported for now. 
+
+### Metricbeat [ingest-metricbeat-limitations]
+
+* The [Secrets Keystore](beats://reference/metricbeat/keystore.md) is not supported. 
+* The [Translate GUID processor](beats://reference/metricbeat/processor-translate-guid.md) is not supported.
+* The [Fingerprint processor](beats://reference/metricbeat/fingerprint.md) does not support the md5 and sha1 method. 
+* The [Community ID Network Flowhash processor](beats://reference/metricbeat/community-id.md) is not supported. 
+* The [Azure module](beats://reference/metricbeat/metricbeat-module-azure.md) is currently not supported. The [Add Cloud Metadata processor](beats://reference/metricbeat/add-cloud-metadata.md) does not support the Azure Virtual Machine provider currently. 
```

Comment:
* The [Google Cloud Platform module](https://www.elastic.co/docs/reference/beats/metricbeat/metricbeat-module-gcp) is currently not supported.